### PR TITLE
Fix form submission (#121)

### LINKED ISSUE #121: Fix form submission issue with Formik

### PR DESCRIPTION
**Pull request by landonreed**
This PR closes #121. Previously, I had tried to encapsulate the Formik logic/behavior in `Form.js` (in case we wanted to swap it out with some other approach), but ultimately just decided for a simpler implementation as recommended by Formik. We could use a higher order component here also, but for this MVP using the `<Formik />` component directly seemed appropriate.

### CODE REVIEW

File: util/index.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+export const submitForm = (path) => async values => {
+    const res = await fetch(
+        // FIXME: use process.env.API_ENDPOINT
+        `http://localhost:5000/forms/${path}`,
```
Comment (sarahgreen) [inline util/index.js]: Is there a reason to not go ahead and use `process.env.API_ENDPOINT` now as opposed to hardcoding in localhost?

**Review by sarahgreen — COMMENTED** [review]
I get this error when trying to run local on this branch. I ran `npm i` and restarted everything. Any suggestions on what to try?

Screen Shot 2021-03-02 at 10 01 40 AM

**Comment by landonreed** [comment]
@sarahgreen, this is a bug with how I'm importing the util function. (Sorry moving too quickly this morning.) Will work on a fix.

**Comment by sarahgreen** [comment]
@landonreed The app compiles for me now! But I still get this error when I try to submit a form locally:
Screen Shot 2021-03-03 at 10 55 23 AM

**Comment by sarahgreen** [comment]
@landonreed The forms no longer error out for me – submission doesn't go to Google Sheets but I imagine that could be a config issue.

**Pull request merged**